docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -265,6 +265,77 @@ While these are out of scope for the present guide,
 that doesn't mean they are less important than race, ethnicity, and ancestry.
 We may address these concepts in a future guide.
 
+Glossary of General Terminology
+-------------------------------
+In this section, you will find a glossary of general terminology and acronyms that will be used 
+throughout this style guide. 
+
+
+.. list-table:: **Glossary**
+   :header-rows: 1
+
+   * - Term
+     - Definition
+   * - Race
+     - A social construct with no inherent biological meaning that categorizes people primarily based on a few visible characteristics such as skin and eye color, hair, and facial features. 
+   * - Ethnicity
+     - Broad cultural group identities. These groups can share language, food, art, religion, social norms, and so on. 
+   * - Ancestry
+     - Broadly, characteristics of one’s ancestors. In this guide we refer to geographic ancestry, where your ancestors are from, or lived approximately 500 years ago.
+   * - Racism
+     - The belief that race influences factors outside of the few visible characteristics listed above (skin color, hair, eye color, etc.). Racism often has been used as a justification to see some people as superior or inferior in some dimension of intelligence, abilities, or qualities. 
+   * - Scientific Racism
+     - An attempt by scientists to prove racist theories in order to justify discrimination and allow for racist systems of inequality to continue. The science used was blatantly incorrect and has been debunked. These racist theories are especially dangerous as they can appear to be from valid sources, even though they are not valid science. 
+   * - Institutional Racism
+     - A form of systematic discrimination, based on race or ethnic group, in which societal policies or practices create an advantage for some groups and harm for others. Also known as structural, societal, or systemic racism.  
+   * - Ethnic Discrimination
+     - The belief that an ethnic group is inherently superior or inferior in some dimension of intelligence, abilities, or qualities. 
```

Comment:
```suggestion
     - The belief that an ethnic group is inherently superior or inferior in some dimension of intelligence, abilities, or other qualities. 
```
Same comment as above: "Qualities" is too vague and includes intelligence and abilities.